core/idotmatrix/common.py

```diff
@@ -113,3 +135,33 @@ def setJoint(self, mode):
             )
         except BaseException as error:
             logging.error("could not change the device joint: {}".format(error))
+            
+    def set_password(self, password: int) -> None:
```

Comment:
Please split the set password stuff into a separate pull request as this is a separate feature :)

Reply:
For larger PRs certainly - however, at this stage of the project I'd rather value clear commit messages over single feature per branch principles (probably a matter of taste :) ). lmk.